### PR TITLE
[miio] Fix for $value$ wrong at 2nd time

### PR DESCRIPTION
**Pull request by marcelrv**
Bug fix for yeelight commands

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java [hunk 1]
```diff
@@ -121,11 +121,12 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 for (MiIoDeviceAction action : miIoBasicChannel.getActions()) {
                     @Nullable
                     JsonElement value = null;
-                    JsonArray parameters = action.getParameters();
+                    JsonArray parameters = action.getParameters().deepCopy();
                     for (int i = 0; i < action.getParameters().size(); i++) {
                         JsonElement p = action.getParameters().get(i);
                         if (p.isJsonPrimitive() && p.getAsString().toLowerCase().contains("$value$")) {
                             valuePos = i;
+                            continue;
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java:129]: Are you sure this is correct? continue here is the last statement in the for loop. So it just continues with the next entry. Did you mean to about the for loop with `break`?

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java:129]: thanks. I think you're right. will fix right away

**Review by Hilbrand — APPROVED** [review]

**Comment by fwolter** [comment]
As spotless is enabled in the `2.5.x` branch now, you need to fix those formatting errors with `mvn spotless:apply`.

**Comment by wborn** [comment]
I think Spotless is happy because I retriggered the Travis build for this one @fwolter. 
For other PRs it fails after retriggering the build.
Example: https://travis-ci.com/github/openhab/openhab-addons/jobs/380401307

**Comment by fwolter** [comment]
I'm wondering how spotless is enabled in the PRs without rebasing?

**Comment by wborn** [comment]
Travis doesn't build the branch but the result of the PR changes merged into 2.5.x.

If you reply the commands Travis executes you can see the result yourself

```
git fetch origin +refs/pull/8399/merge
git checkout -qf FETCH_HEAD
```

```
$ git log

commit 9fa90cc3fbe2f0463a2f94a4fdcbdbb9d1b867e8 (HEAD)
Merge: a270da6c42 e945a86abf
Author: Marcel <marcelrv@users.noreply.github.com>
Date:   Sat Sep 5 12:52:27 2020 +0200

    Merge e945a86abfc04784c1621b96b21b207977ca23cb into a270da6c42b7c1e31c66c7b73fe7a7de4d762b48

commit a270da6c42b7c1e31c66c7b73fe7a7de4d762b48
Author: Wouter Born <github@maindrain.net>
Date:   Sat Sep 5 12:51:47 2020 +0200

    Fix broken build (#8402)
    
    PR #8383 was created before Spotless was enforced.
    
    Signed-off-by: Wouter Born <github@maindrain.net>
```

**Comment by fwolter** [comment]
That makes sense, thanks!

**Pull request merged**